scalastyle-config.xml

```diff
@@ -0,0 +1,117 @@
+<scalastyle>
+ <name>Scalastyle standard configuration</name>
+ <check level="warning" class="org.scalastyle.file.FileTabChecker" enabled="true"></check>
```

Comment:
Could you please update as below as it is becoming a tag with empty body? Same for other places at line 28, 29,30,52, 68-75, 91,92,114-116?

 <check level="warning" class="org.scalastyle.file.FileTabChecker" enabled="true"/>